src/Card/CardBody.js

```diff
@@ -8,7 +9,7 @@ const CardBody = props => {
     }
 
     return (
-        <div >
+        <Wrapper >
```

Comment:
You can use `<React.Fragment>` build-in HOC of the lib if you need to just wrap up several elements. Or, now blank tag also becomes very popular `<></>`